bindings/py/cpp_src/plugin/unittest/PyRegionTest.cpp

```diff
@@ -75,36 +74,6 @@ using namespace nupic;
 
 bool verbose = false;
 
-struct MemoryMonitor {
```

Comment:
👍 agreed because gtest framework cannot be used to do memory leak tests in this way.